### PR TITLE
Docs 11996 what not imported bsj

### PR DESCRIPTION
**Pull request by bjoson-mulesoft**

### CODE REVIEW

**Review by fedebrasburg — DISMISSED** [review]

**Review by fedebrasburg — DISMISSED** [review]

**Review by nledesma — COMMENTED** [review]
LGTM! 🚀 Lets wait for @fedebrasburg  approval, who already gave feedback

EDIT: just approved it seconds ago, hehe

File: modules/ROOT/pages/import-api-task.adoc
Comment (dhirsch1) [inline modules/ROOT/pages/import-api-task.adoc]: Suggtion: "by importing it to..."

**Review by dhirsch1 — DISMISSED** [review]
One suggestion

File: modules/ROOT/pages/import-api-task.adoc
Comment (bjoson-mulesoft) [reply modules/ROOT/pages/import-api-task.adoc]: Done.

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**
